static/app/views/onboarding/targetedOnboarding/onboarding.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import {AnimatePresence, useAnimation} from 'framer-motion';
+
+import Hook from 'sentry/components/hook';
+import LogoSentry from 'sentry/components/logoSentry';
+import SentryDocumentTitle from 'sentry/components/sentryDocumentTitle';
+import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
+import PageCorners from 'sentry/views/onboarding/components/pageCorners';
+
+import TargetedOnboardingWelcome from './welcome';
+
+export default function Onboarding() {
+  const Contents = () => {
+    const cornerVariantControl = useAnimation();
+    const updateCornerVariant = () => {
+      cornerVariantControl.start('top-right');
+    };
+
+    // XXX(epurkhiser): We're using a react hook here becuase there's no other
+    // way to create framer-motion controls than by using the `useAnimation`
+    // hook.
+
+    React.useEffect(updateCornerVariant, []);
+
+    return (
+      <Container>
+        <AnimatePresence exitBeforeEnter onExitComplete={updateCornerVariant}>
+          <TargetedOnboardingWelcome />
+        </AnimatePresence>
+        <PageCorners animateVariant={cornerVariantControl} />
+      </Container>
+    );
+  };
```

Comment:
This only needed to be in it's own component before bc the `useAnimation` couldn't be used in a classic React.Component class.